### PR TITLE
UIFix for Debrief Conclusion

### PR DESCRIPTION
**Pull request by NgaiJustin**
## Overview

Fix UI for debriefConclusion

## Changes Made

- Add Overlay for for button
- Fix UI layout 

## Test Coverage

- iPhone XR
- iPhone 13 (Simulator)

## Related PRs or Issues (delete if not applicable)

- Merge #101  first
- #100 

## Screenshots (delete if not applicable)

<table>
  <tr>
     <td>Before</td>
     <td>After</td>
  </tr>
  <tr>
    <td>
Screenshot 2022-04-21 at 5 23 20 PM
</td>
    <td>
Screenshot 2022-04-21 at 5 22 48 PM
</td>
  </tr>
</table>

### CODE REVIEW

**Review by dishanest — APPROVED** [review]
Looks good! One note: there seems to be many files changed because this branch depends on another branch PR that has not been merged with `master`. I think a solution to this is to mark this PR as a draft until its dependent branch is merged, so that reviewers can see exactly what you did in _this_ branch rather than both branches.

File: Volume/Views/WeeklyDebriefView.swift [hunk 1]
```diff
@@ -98,6 +101,12 @@ struct WeeklyDebriefView: View {
                     .foregroundColor(.volume.orange)
                     .font(.helveticaBold(size: 16))
             }
+            .padding(.horizontal, 20)
+            .padding(.vertical, 10)
+            .overlay(RoundedRectangle(cornerRadius: 26)
+                .stroke(Color.volume.orange, lineWidth: 2))
+
+            Spacer().frame(height: 100)
```
Comment (Npikielny) [inline Volume/Views/WeeklyDebriefView.swift:109]:
```suggestion
            Spacer()
                .frame(height: 100)
```

File: Volume/Views/WeeklyDebriefView.swift [hunk 2]
```diff
@@ -98,6 +101,12 @@ struct WeeklyDebriefView: View {
                     .foregroundColor(.volume.orange)
                     .font(.helveticaBold(size: 16))
             }
+            .padding(.horizontal, 20)
+            .padding(.vertical, 10)
+            .overlay(RoundedRectangle(cornerRadius: 26)
+                .stroke(Color.volume.orange, lineWidth: 2))
```
Comment (Npikielny) [inline Volume/Views/WeeklyDebriefView.swift:107]: @dishanest wudya think?
```suggestion
            .overlay(
                RoundedRectangle(cornerRadius: 26)
                    .stroke(.volume.orange, lineWidth: 2)
            )
```
Also, not sure if we can implicitly get from Color ¯\_(ツ)_/¯

**Review by Npikielny — APPROVED** [review]
Looks good!

**Review by emarc0314 — APPROVED** [review]
👍

**Pull request merged**